kroxylicious-operator/examples/simple/02.server-certificate.yaml

```diff
@@ -0,0 +1,33 @@
+#
+# Copyright Kroxylicious Authors.
+#
+# Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+#
+
+---
+apiVersion: cert-manager.io/v1
+kind: Certificate
+metadata:
+  name: server-certificate
+  namespace: my-proxy
+spec:
+  isCA: true
+  commonName: my-cluster-bootstrap.my-proxy.svc.cluster.local
```

Comment:
IIRC modern usage is to omit commonName, and use SAN for make all the DNS names (i.e. add it to `dnsNames`?)

Reply:
From a quick read around, it seems the general advice is to have a non null commonName for reasons of compatibility with cert managers that may rely on it.   I agree Kroxylicious shouldn't rely on it being present